patternlab/styleguide/source/assets/js/modules/feedbackForm.js

```diff
@@ -0,0 +1,66 @@
+export default (function (window, document, $) {
+
+  const $feedbackForm = $('.ma__mass-feedback-form__form');
+  const $fields = $('.ma__mass-feedback-form__fields');
+  const $textArea = $fields.find('textarea');
+  const $contactForm = $feedbackForm.find('.ma__mass-feedback-form__contact-form');
+  const $loadRadios = $feedbackForm.find('.feedback-load input[type="radio"]');
+  const $contactRadios = $feedbackForm.find('.user-response__contact input[type="radio"]');
+  const $Response = $('.ma__mass-feedback-form__form--user-response');
+  const $noResponseLink = $Response.find('.form--no-response');
+  const $alertToggle = $('.ma__header-alert__hide');
+
+  // Open no reponse answer accordion.
+  $noResponseLink.on('click', function () {
+    $Response.toggleClass('is-open');
+  });
+
+  // Auto size the text boxes
+  $textArea.each(function () {
+    let $el = $(this);
+
+    $el.on('keyup', function () {
+      if ($el.prop('scrollHeight') > $el.prop('clientHeight')) {
+        $el.css('height', $el.prop('scrollHeight') + "px");
+      }
+      else {
+        $el.css('height', "100%");
+      }
+    });
+  });
+
+  // Open feedback form on radio button selection.
+  $loadRadios.on('change', function () {
+
+    let foundIndicator = $("input[name='foundIndicator']:checked").val();
```

Comment:
As we discussed, we need to be able to change the radio name.  This assumes it will be 'foundIndicator'.